File: app/src/main/java/in/woloo/www/trendingblog/epoxy/TrendingBlogTextModel.java
package in.woloo.www.trendingblog.epoxy;

import android.widget.TextView;

import androidx.annotation.NonNull;

import com.airbnb.epoxy.EpoxyAttribute;
import com.airbnb.epoxy.EpoxyModelClass;
import com.airbnb.epoxy.EpoxyModelWithHolder;

import butterknife.BindView;
import in.woloo.www.R;
import in.woloo.www.base.BaseEpoxyHolder;

@EpoxyModelClass(layout = R.layout.model_title)
public abstract class TrendingBlogTextModel extends EpoxyModelWithHolder<TrendingBlogTextModel.Holder> {
    @EpoxyAttribute
    String title;

    @Override
    public void bind(@NonNull Holder holder) {
        holder.tvTitle.setText(title);
    }

    static class Holder extends BaseEpoxyHolder {
        @BindView(R.id.tvTitle)
        TextView tvTitle;
    }
}

File: app/src/main/java/in/woloo/www/more/trendingblog/epoxy/TrendingBlogTextModel.kt
package `in`.woloo.www.more.trendingblog.epoxy

import com.airbnb.epoxy.EpoxyAttribute
import com.airbnb.epoxy.EpoxyModelClass
import com.airbnb.epoxy.EpoxyModelWithHolder
import `in`.woloo.www.R
import `in`.woloo.www.application_kotlin.base.BaseEpoxyHolder

@EpoxyModelClass(layout = R.layout.model_title)
abstract class TrendingBlogTextModel : EpoxyModelWithHolder<TrendingBlogTextModel.Holder>() {
    @JvmField
    @EpoxyAttribute
    var title: String? = null
    override fun bind(holder: Holder) {
        /*holder.tvTitle.setText(title)*/
    }

    class Holder : BaseEpoxyHolder() { /*  @BindView(R.id.tvTitle)
        TextView tvTitle;*/
    }
}
